.github/workflows/register-buildpack.yml

```diff
@@ -0,0 +1,260 @@
+name: Register Buildpack
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  register:
+    env:
+      NAMESPACE_REPO: registry-namespaces
+      GITHUB_OWNER: buildpacks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node Dependencies
+        working-directory: .github/workflows/scripts/validate
+        shell: bash
+        run: |
+          npm install
+      - id: detect_validate
+        name: Validate Buildpack Issue
+        uses: actions/github-script@v2
+        with:
+          result-encoding: string
+          script: |
+            const path = require('path')
+            const scripts = require(path.resolve('.github/workflows/scripts/validate/index.js'))
+            let result
+            try {
+              result = scripts.validateIssue({context})
+            } catch (error) {
+              console.error(error)
+              process.exit(1)
+            }
+            console.log(`::set-env name=BUILDPACK::${JSON.stringify(result)}`)
+
+      - uses: actions/github-script@v2
+        name: Retrieve Owners
+        with:
+          github-token: ${{ secrets.BOT_TOKEN }}
+          script: |
+            const buildpackInfo = JSON.parse(process.env.BUILDPACK)
+            let registryOwners = ''
+            try {
+              const {data} = await github.repos.getContent({
+                owner: process.env.GITHUB_OWNER,
+                path: `v1/${buildpackInfo.ns}.json`,
+                repo: process.env.NAMESPACE_REPO
+              })
+              const buff = new Buffer.from(data.content, 'base64')
+              registryOwners = buff.toString('utf-8')
+
+            } catch (error) {
+              if (error.status && error.status === 404) {
+                console.error('Creating file since it does not exist')
+                const content = {
+                  owners: [
+                    {
+                      id: context.payload.sender.id,
+                      type: 'github_user'
+                    }
+                  ]
+                };
+                const buff = Buffer.from(JSON.stringify(content), 'utf-8');
+                registryOwners = buff.toString('utf-8')
+
+                await github.repos.createOrUpdateFile({
+                  owner: process.env.GITHUB_OWNER,
+                  repo: process.env.NAMESPACE_REPO,
+                  path: `v1/${buildpackInfo.ns}.json`,
+                  message: 'initial commit',
+                  content: buff.toString('base64'),
+                  committer: {
+                    name: process.env.GITHUB_OWNER,
+                    email: 'longoria.public@gmail.com'
+                  },
+                  author: {
+                    name: process.env.GITHUB_OWNER,
+                    email: 'longoria.public@gmail.com'
```

Comment:
Can we change commit email addresses to the bot's:  `cncf-buildpacks-maintainers@lists.cncf.io`?